pages/_app.tsx

```diff
@@ -13,6 +13,7 @@ const pages = [
   { label: "Projektit", path: "/projects" },
   { label: "Tietoa meistä", path: "/about" },
   //{ label: "Tietoa", path: "/about-us" }
+  { label: "Koneet kiertoon", path: "/computer-sharing" }
```

Comment:
Update this to `/koneet-kiertoon`, as the path was updated in https://github.com/Testausserveri/testausserveri.fi/pull/99/commits/0fe39eecc43410bb1f9ba8ed1ace1f8f226f4211

Reply:
Fixed in [e4125c5](https://github.com/Testausserveri/testausserveri.fi/pull/99/commits/e4125c56cf8cab9d9ec0c55e15c9f1b8876d5c51)